workflows/diagnostics/fv3net/diagnostics/offline/_input_sensitivity.py

```diff
@@ -0,0 +1,215 @@
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+import logging
+from typing import Tuple, Hashable, Iterable, Dict
+import xarray as xr
+
+import fv3fit
+from fv3fit.sklearn._random_forest import SklearnWrapper
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
```

Comment:
The majority of this diff is the same as was moved out of `offline._helpers`. The only new parts are `plot_input_sensitivity` and `dataset_to_dict_input`.